wherehows-web/app/templates/components/datasets/containers/health-score-gauge.hbs

```diff
@@ -1,8 +1,14 @@
 {{#if getHealthScoreTask.isIdle}}
-  {{visualization/charts/score-gauge
+  {{#visualization/charts/score-gauge
     class="dataset-health-score"
     score=healthScore
-    title="Health Score:"}}
+    customTitleTemplate=true}}
```

Comment:
You don't need this `customTitleTemplate` attribute since it is actually contrived, you can take advantage of the HTMLBars global `hasBlock` and just rewrite the template below with:

```hbs
{{#if hasBlock}}
  {{yield}}
{{else}}
   ...
{{/if}}
```
this way you're not limiting the use case to just a custom title but also future proofing the component for other potential use cases